tests/unit_tests/test_sda_flight_boundary.py

```diff
@@ -0,0 +1,25 @@
+from matplotlib.path import Path
+from SDA import*
+import unittest
+import numpy as np
+
+class FlightBoundaryTestCase(unittest.TestCase):
+
```

Comment:
This is a very important class. Please add additional tests to ensure that the path works in a complex, irregular polygon. You need to test paths that will be edge cases (right outside of the path, right inside, in a concave place, etc)